core/include/thread.h

```diff
@@ -184,9 +184,13 @@ struct _thread {
                                          flags                          */
 #endif
 #if defined(MODULE_CORE_MSG)
-    list_node_t msg_waiters;        /**< threads waiting on message     */
-    cib_t msg_queue;                /**< message queue                  */
-    msg_t *msg_array;               /**< memory holding messages        */
+    list_node_t msg_waiters;        /**< threads waiting for their message
+                                         to be delivered to this thread
+                                         (i.e. all blocking sends)      */
```

Comment:
I think "all blocked sends" would be more exact, because a blocking `msg_send()` call may deliver its message to a potential queue directly without ending up in this list.